index.html

```diff
@@ -5155,6 +5155,56 @@ <h2>The <code>application/credential+ld+json</code> Media Type</h2>
 
 </section>
 
+<section id="presentation-media-type">
+  <h2>The <code>application/presentation+ld+json</code> Media Type</h2>
+  <p>
+    This specification registers the <code>application/presentation+ld+json</code> MIME Media Type specifically for
```

Comment:
```suggestion
    This specification registers the <code>application/presentation+ld+json</code> Media Type specifically for
```

IETF guidance is to not call them MIME (Multipurpose Internet Mail Extensions) Types anymore since the original concept grew past the goal of use in "internet mail" into other things like HTTP... From the IANA Media Types Registry -- "[[RFC2046](https://www.iana.org/go/rfc2046)] specifies that Media Types (formerly known as MIME types) and Media Subtypes will be assigned and listed by the IANA."